tests/models/seamless_m4t_v2/test_modeling_seamless_m4t_v2.py

```diff
@@ -784,7 +790,7 @@ def update_generation(self, model):
 
         model.generation_config = generation_config
 
-    def prepare_text_input(self):
+    def prepare_text_input(self, is_rus=False):
```

Comment:
Why not just make this `tgt_lang` and then you can easily test with different target languages both in an outside of the supported languages with `generate_speech` as `True` and `False`?